File: Online-Ticketing/src/main/java/com/ticketing/application/utils/AuthResponse.java
package com.ticketing.application.utils;

import javax.persistence.Entity;
import javax.persistence.Table;

import org.springframework.web.bind.annotation.ResponseBody;

import com.ticketing.application.model.*;

@ResponseBody
public class AuthResponse {
    
	Boolean result;
    String message;
    
	public AuthResponse(Boolean result, String message) {
		super();
		this.result = result;
		this.message = message;
	}
    
    
}

File: frontend/src/app/utils/AuthResponse.ts
import {User} from '../auth/user';

export class AuthResponse{

    public result:boolean;
    public user:User;
    public message:string;
    

    constructor(result:boolean, user:User, message:string){
        this.result = result;
        this.user = user;
        this.message=message;
    }
}